client/ayon_core/pipeline/workfile/workfile_template_builder.py

```diff
@@ -1620,6 +1617,20 @@ def _get_representations(self, placeholder):
                 )
             }
 
+        elif builder_type == "context_folder":
+            folder_ids = [current_folder_entity["id"]]
+
+        elif builder_type == "linked_folders":
+            # Get all linked folders for the current folder
+            if hasattr(self, "builder") and isinstance(
+                    self.builder, AbstractTemplateBuilder):
```

Comment:
Why is this `hasattr` and check needed? In what case is the builder not available?

Reply:
IDEs don't like it. It is mixin/interface meant for `PlaceholderItem` which does implement the `builder` attribute.